### PR TITLE
Feat optimize sfs

### PR DESCRIPTION
**Pull request by deer-hang**

### CODE REVIEW

File: deploy/sfs-csi-plugin/kubernetes/csi-sfs-controller.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ spec:
             - "--csi-address=$(ADDRESS)"
             - "--enable-leader-election"
             - "--leader-election-type=leases"
+            - "--timeout=3m"
```
Comment (chengxiangdong) [inline deploy/sfs-csi-plugin/kubernetes/csi-sfs-controller.yaml:33]: Line 40: csi-attacher also needs to append arg: `-"--timeout=3m"`.

File: pkg/sfs/utils.go [hunk 2]
```diff
@@ -103,29 +26,17 @@ func validateCreateVolumeRequest(req *csi.CreateVolumeRequest) error {
...
-// (2 GiB is the smallest allocatable volume that can hold 1500MiB)
-func RoundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
-	roundedUp := volumeSizeBytes / allocationUnitBytes
-	if volumeSizeBytes%allocationUnitBytes > 0 {
-		roundedUp++
-	}
-	return roundedUp
-}
-
+// TODO after feat-sfs-idempotent merged. Delete this method
```
Comment (chengxiangdong) [inline pkg/sfs/utils.go:39]: do you mean remove `bytesToGiB`?  i see it is used in controller

File: pkg/sfs/controllerserver.go [hunk 3]
```diff
@@ -40,38 +40,38 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
...
+	}
 
 	requestedSize := req.GetCapacityRange().GetRequiredBytes()
-    if requestedSize == 0 {
-        // At least 1GiB
-        requestedSize = 1 * bytesInGiB
-    }
+	if requestedSize == 0 {
+		// At least 1GiB
+		requestedSize = 1 * bytesInGiB
```
Comment (chengxiangdong) [inline pkg/sfs/controllerserver.go:51]: minimum capacity changed to 10 GB

File: deploy/sfs-csi-plugin/kubernetes/csi-sfs-controller.yaml (hunk 1 above)
Comment (deer-hang) [reply deploy/sfs-csi-plugin/kubernetes/csi-sfs-controller.yaml:33]: done

File: pkg/sfs/utils.go (hunk 2 above)
Comment (deer-hang) [reply pkg/sfs/utils.go:39]: done

File: pkg/sfs/controllerserver.go (hunk 3 above)
Comment (deer-hang) [reply pkg/sfs/controllerserver.go:51]: this branch is used to change sfs package utils and config. "10GB" will be changed in branch feat-sfs-idempotent

**Pull request merged**